scripts/InstancingWithModsSystem.js

```diff
@@ -0,0 +1,129 @@
+class InstancingSystem extends System {
+
+    // todo - title it InstancingWithModsSystem
+
+    constructor() {
+        super();
+
+        console.log(this.componentName);
+
+        const this.instanceCount = 5;
+
+        // DataOrientedDesign arrays
+        let this.positions = null;
+        let this.scales = null;
+        let this.rotations = null;
+        let this.meshes = [];
+        let this.geometry = null;
+        console.log('constructed an instancing system');
+    }
+
+    update(deltaTime, frame) {
+        for(const entity of this.registry){
+            switch (entity.instancing?.type) {
```

Comment:
this line should be entity.components.get('instancing')?.type

Reply:
√